style.css

```diff
@@ -0,0 +1,423 @@
+@font-face {
+  font-family: Fraunces-font;
+  src: url(./assets/Fraunces-VariableFont_SOFT\,WONK\,opsz\,wght.ttf);
+}
+
+@font-face {
+  font-family: Fraunces-font-bold;
+  src: url(./assets/static/Fraunces_72pt\,Soft-Bold.ttf);
+}
+
+footer {
+  color: gray;
+}
+
+.date {
+  color: white;

```

Comment:
- [ ] you can use a bootstrap class for this color https://getbootstrap.com/docs/4.0/utilities/colors/